process/transactions/scrsDataToTransactions.go

```diff
@@ -48,6 +48,11 @@ func (st *scrsDataToTransactions) attachSCRsToTransactionsAndReturnSCRsWithoutTx
 
 func (st *scrsDataToTransactions) addScResultInfoIntoTx(dbScResult *data.ScResult, tx *data.Transaction) {
 	tx.SmartContractResults = append(tx.SmartContractResults, dbScResult)
+	if isRelayedTx(tx) && len(tx.SmartContractResults) == 1 {
```

Comment:
can you put this condition into a variable with an appropriate name?

Reply:
done.